pkg/resources/tag_masking_policy_association.go

```diff
@@ -208,23 +208,23 @@ func ReadTagMaskingPolicyAssociation(d *schema.ResourceData, meta interface{}) e
 	if err != nil {
 		return err
 	}
+	/*
```

Comment:
why is it commented (same for the code below)

Reply:
it was causing an acc test to fail, and anyways its not really needed since it isn't a computed attribute